examples/express/package.json

```diff
@@ -1,25 +1,26 @@
 {
-    "name": "express-basic",
-    "version": "0.1-alpha",
-    "private": true,
-    "type": "module",
-    "scripts": {
-        "start": "nodemon -e ts -w ./src -x npm run watch:serve",
-        "watch:serve": "node --loader ts-node/esm index.ts",
-        "dev": "tsx index.ts"
-    },
-    "dependencies": {
-        "@edgestore/server": "^0.1.5-alpha.1",
-        "nodemon": "^3.0.1",
-        "zod": "^3.22.4"
-    },
-    "devDependencies": {
-        "@types/express": "^4.17.21",
-        "@types/node": "^18.18.10",
-        "eslint": "^8.40.0",
-        "eslint-config-next": "^13.4.9",
-        "express": "^4.18.2",
-        "tsx": "^4.1.4",
-        "typescript": "^5.2.2"
-    }
+  "name": "express",
+  "version": "0.1.5-alpha.1",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "nodemon -e ts -w ./src -x npm run watch:serve",
+    "watch:serve": "node --loader ts-node/esm src/index.ts",
+    "dev": "dotenv -e .env.local -- tsx src/index.ts"
+  },
+  "dependencies": {
+    "@edgestore/server": "^0.1.5-alpha.1",
+    "express": "^4.18.2",
```

Comment:
I think express should be a dependency instead of a devDependency, so I moved it.